tests/controllers/users.test.js

```diff
@@ -0,0 +1,146 @@
+const chai = require("chai");
+const sinon = require("sinon");
+chai.use(require('sinon-chai'));
+const request = require("supertest");
+const { expect } = chai;
+var assert = require('assert');
+
+const { app } = require("../../startup");
+const db = require("../../models");
+const sendEmail = require("../../helpers/email");
+
+describe("GET users", async () => {
+  beforeEach(async () => {
+    await db.User.destroy({ truncate: { cascade: true } });
+    await db.User.create({
+      firstName: "Calumn",
```

Comment:
Let's use this lib `https://fakerjs.dev/guide/` to avoid you generating random data.

With this lib you can say `firstName: faker.name.firstName()`

Reply:
how do i test my filters then?

Reply:
Don't create users before each test. Instead of that, create users for the specific test.

Reply:
Or create "global" variable:
```
let user_1 = null;
let user_2 = null;
```

and then inside `beforeEach` set user_1 and user_2

but it's better for tests to be isolated.

You can create helper method to generate user with default values if not provided:

```
function makeUser(data):
   const { firstName = faker.firstName, ... } = data;
```